doc/man3/ERR_set_mark.pod

```diff
@@ -2,15 +2,16 @@
 
 =head1 NAME
 
-ERR_set_mark, ERR_pop_to_mark - set marks and pop errors until mark
+ERR_set_mark, ERR_clear_last_mark, ERR_pop_to_mark
+- set marks, clear mark and pop errors until mark
```

Comment:
Singular/plural: _marks_ / _mark_.

I'd probably change the _marks_ to _mark_.